``` shell
C:\conda\conda-bld\work\spatialindex-src-1.8.5\src\spatialindex\LineSegment.cc(488): warning C4273: 'SpatialIndex::operator <<': inconsistent dll linkage 
c:\conda\conda-bld\work\spatialindex-src-1.8.5\include\spatialindex\LineSegment.h(86): note: see previous definition of '<<' 
NMAKE : fatal error U1077: 'C:\PROGRA~2\MI0E91~1.0\VC\bin\cl.exe' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\nmake.exe"' : return code '0x2'
Stop.
Command failed: C:\Windows\system32\cmd.exe /c call bld.bat
Command exited with code 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,11 +9,14 @@ env:
     - secure: "iza+WzpX+bG4SOfrz425zTdj2CFz0rjcgR3a5LGqwAHMKRf9wGOFsS0/e313pMFPOMEno9bXpKkDoL7dPkkpTsg81aq2PViQAntmS7DdbVGrQD7of4254rvuA3gaZEYYOADL+6PCOV/HOaN/+nDnQrr/KwEVBK8QqsDNEh+allG1oLDNXzCovv5TjDo+FbuixRgUhaPsNwX7m9iK+Wp99zXi9FBkNQVZZ0rpsdCJ5LWVL267ZfD2eLd0ebjMTVAq4u14dUsuIzd7RE55IZYxbTTJORSbM9Rv3nWJXzvJkxNGy7knhAh/lkMC31JPaZXgotNkhIBVGrecuCpBb5w3s+GOKT/DZrquboIxC7/kWNOwVIEq1tyAijIZ365A5N2QX8eiDrH0ng6wpKOq8mGjrhp0pom3eYqF0TYs/7BhNbhMLUc/8flBfzUZ/XqjcfghyrLNzCVoJVoFkK/3V/4YGQEe1zQOL1SHFWX0aqazqiUpTfzVobnAEbCQNJRGy4vBgy9NZE0BF1PTPvWSHdJWkPCIt3/mjMP2uSPxTsXnzpqzeqqHJ80qgS9qbpVHr0j4rVvcgBSm2nhznCC40aY4STDCLiZ+7TpjYwzFGkKPBT46e78Y3gsbfaKstwxsNgQhtMBLkq+a454jCji6/s+TvxPtKUKHo/x0Dw2zAdWvMsQ="
 
 
+before_install:
+    - brew remove --force $(brew list)
+
 install:
     - |
       MINICONDA_URL="http://repo.continuum.io/miniconda"
       MINICONDA_FILE="Miniconda3-latest-MacOSX-x86_64.sh"
-      wget "${MINICONDA_URL}/${MINICONDA_FILE}"
+      curl -O "${MINICONDA_URL}/${MINICONDA_FILE}"
       bash $MINICONDA_FILE -b
 
       export PATH=/Users/travis/miniconda3/bin:$PATH

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -30,6 +30,11 @@ environment:
     - TARGET_ARCH: x64
       CONDA_PY: 34
 
+    - TARGET_ARCH: x86
+      CONDA_PY: 35
+    - TARGET_ARCH: x64
+      CONDA_PY: 35
+
 
 # We always use a 64-bit machine, but can build x86 distributions
 # with the TARGET_ARCH variable.

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -1,10 +1,8 @@
-mkdir build
-cd build
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      .
+if errorlevel 1 exit 1
 
-set LIB=%LIBRARY_LIB%;%LIB%
-set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
-
-cmake .. -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release
-
-nmake install || exit 1
+nmake install
+if errorlevel 1 exit 1

--- a/recipe/cmakelists.patch
+++ b/recipe/cmakelists.patch
@@ -1,0 +1,20 @@
+--- CMakeLists.txt.orig	2016-04-09 14:40:16.940122700 +1000
++++ CMakeLists.txt	2016-04-09 15:18:56.564971600 +1000
+@@ -42,7 +42,7 @@
+ 
+ if(WIN32)
+   if (MSVC)
+-    if (MSVC80 OR MSVC90 OR MSVC10 OR MSVC11 OR MSVC12 OR MSVC13)
++    if (MSVC80 OR MSVC90 OR MSVC10 OR MSVC11 OR MSVC12 OR MSVC13 OR MSVC14)
+         if( CMAKE_SIZEOF_VOID_P EQUAL 8 )
+             set( SIDX_LIB_NAME "spatialindex-64" )
+             set( SIDX_C_LIB_NAME "spatialindex_c-64" )
+@@ -131,7 +131,7 @@
+ 
+ if(WIN32)
+   if (MSVC)
+-    if (MSVC80 OR MSVC90 OR MSVC10 OR MSVC11 OR MSVC12 OR MSVC13)
++    if (MSVC80 OR MSVC90 OR MSVC10 OR MSVC11 OR MSVC12 OR MSVC13 OR MSVC14)
+       add_definitions(-D_CRT_SECURE_NO_DEPRECATE)
+       add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+       add_definitions(-D_CRT_NONSTDC_NO_WARNING)

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -8,25 +8,25 @@ source:
     fn: spatialindex-src-{{ version }}.tar.gz
     url: http://download.osgeo.org/libspatialindex/spatialindex-src-{{ version }}.tar.gz
     md5: 4065c6218ce3d1c4906beb3a313470db
+    patches:
+        - tool.h.patch  # [win and py35]
+        - cmakelists.patch  # [win and py35]
 
 build:
-    skip: True  # [py35 and win]
     number: 1
     features:
         - vc9  # [win and py27]
         - vc10  # [win and py34]
         - vc14  # [win and py35]
 
-
 requirements:
     build:
-        # The VS version is important.
         - python  # [win]
 
 test:
     commands:
-        - exit $(test -f ${PREFIX}/lib/libspatialindex_c.so)  # [linux]
-        - exit $(test -f ${PREFIX}/lib/libspatialindex_c.dylib)  # [osx]
+        - test -f ${PREFIX}/lib/libspatialindex_c.so  # [linux]
+        - test -f ${PREFIX}/lib/libspatialindex_c.dylib  # [osx]
 
 about:
     home: http://libspatialindex.github.io

--- a/recipe/tool.h.patch
+++ b/recipe/tool.h.patch
@@ -1,0 +1,20 @@
+--- include\spatialindex\tools\Tools.h.orig	2016-04-09 14:41:05.660674700 +1000
++++ include\spatialindex\tools\Tools.h	2016-04-09 15:28:33.439971600 +1000
+@@ -28,7 +28,7 @@
+ #pragma once
+ 
+ 
+-#if (defined _WIN32 || defined _WIN64 || defined WIN32 || defined WIN64) && !defined __GNUC__
++#if (defined _WIN32 || defined _WIN64 || defined WIN32 || defined WIN64) && (defined _MSC_VER) && (_MSC_VER < 1900) && !defined __GNUC__
+   typedef __int8 int8_t;
+   typedef __int16 int16_t;
+   typedef __int32 int32_t;
+@@ -46,7 +46,7 @@
+ #endif
+ 
+ #if (defined _WIN32 || defined _WIN64 || defined WIN32 || defined WIN64) && !defined __GNUC__
+-  #ifdef SPATIALINDEX_CREATE_DLL
++  #ifdef SIDX_DLL_EXPORT
+     #define SIDX_DLL __declspec(dllexport)
+   #else
+     #define SIDX_DLL __declspec(dllimport)

